test/controllers/CashAccountV2ControllerSpec.scala

```diff
@@ -38,7 +38,6 @@ import play.api.i18n.{Messages, MessagesApi}
 import play.api.inject.bind
```

Comment:
You have lines over the char limit on line 20 and 284 as well as noted below. I know these aren't thing you have written/or added but might be worth fixing them whilst we're editing the file anyway